flight/Modules/Stabilization/stabilization.c

```diff
@@ -390,16 +591,29 @@ static void stabilizationTask(void* parameters)
 	FlightStatusData flightStatus;
 	SystemSettingsAirframeTypeOptions airframe_type;
 
+	volatile bool settings_flag = true;
+	volatile bool flightstatus_flag = true;
```

Comment:
Suggest, _updated, flag is very ambiguous.

Reply:
OK i just moved these didn't I? :P  But fine :P